### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/6a0ee6caca1e74ff26f78d1d841e3da768346afc

### CODE REVIEW

File: en/brands.ftl [hunk 1]
```diff
@@ -11,7 +11,6 @@
 -brand-name-apple = Apple
 -brand-name-creative-commons = Creative Commons
 -brand-name-facebook = Facebook
--brand-name-github = GitHub
```
Comment (alexgibson) [inline en/brands.ftl:14]: So this is an interesting wrinkle. These brand terms we're adding in https://github.com/mozilla-l10n/www-l10n/pull/41, but have not yet been merged to bedrock in https://github.com/mozilla/bedrock/pull/8958

Comment (pmac) [reply en/brands.ftl:14]: Yeah. Modifying these files directly in this repo and not in bedrock will result in this. Bedrock is meant to be the source of truth for the en files I believe.

Comment (pmac) [reply en/brands.ftl:14]: In future I would update this file in a separate PR in bedrock and let the automation open the PR like #41 .

Comment (alexgibson) [reply en/brands.ftl:14]: Ok, let's close this PR and add directly to bedrock

**Pull request closed without merging**